src/main/java/com/deblock/jsondiff/matcher/StrictJsonObjectPartialMatcher.java

```diff
@@ -0,0 +1,40 @@
+package com.deblock.jsondiff.matcher;
+
+import com.deblock.jsondiff.diff.JsonDiff;
+import com.deblock.jsondiff.diff.JsonObjectDiff;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+
+public class StrictJsonObjectPartialMatcher implements PartialJsonMatcher<ObjectNode> {
+
+    @Override
+    public JsonDiff jsonDiff(Path path, ObjectNode expectedJson, ObjectNode receivedJson, JsonMatcher jsonMatcher) {
+        final var jsonDiff = new JsonObjectDiff(path);
+
+        expectedJson.fields()
+            .forEachRemaining(entry -> {
+                final var expectedPropertyName = entry.getKey();
+                final var expectedValue = entry.getValue();
+                final var receivedValue = receivedJson.get(expectedPropertyName);
+
+                if (receivedValue == null) {
+                    jsonDiff.addNotFoundProperty(expectedPropertyName, expectedValue);
+                } else {
```

Comment:
🤔 
with this code, I think that this example will add a not found property .
received:
```
{
    "property": null
}
```

expected: 
```
{
    "property": "value"
}
```